frontend/src/app/projects/project-detail/model-overview/model-complexity-badge/model-complexity-badge.component.ts

```diff
@@ -70,7 +70,10 @@ export class ModelComplexityBadgeComponent implements OnChanges {
   }
 
   openModelComplexityBadgeDocs() {
-    const docsURL = environment.docsURL || '/docs';
-    window.open(docsURL + '/projects/models/complexity_badge', '_blank');
+    let docsURL = '/docs';
+    if ('docsURL' in environment) {
+      docsURL = environment.docsURL as string;
+    }
```

Comment:
The original line should do the same, right? If docsURL is not defined, it returns `undefined`, which will trigger the or-lhs clause: `/docs`.